README.md

```diff
@@ -1,105 +1,34 @@
-<p align="center">
-  <a href="https://github.com/actions/typescript-action/actions"><img alt="typescript-action status" src="https://github.com/actions/typescript-action/workflows/build-test/badge.svg"></a>
-</p>
+# Terraform Register Module Action
 
-# Create a JavaScript Action using TypeScript
+Use this action to register a repository with a Terraform Cloud registry.
 
-Use this template to bootstrap the creation of a TypeScript action.:rocket:
+## Usage
 
-This template includes compilation support, tests, a validation workflow, publishing, and versioning guidance.  
-
-If you are new, there's also a simpler introduction.  See the [Hello World JavaScript Action](https://github.com/actions/hello-world-javascript-action)
-
-## Create an action from this template
-
-Click the `Use this Template` and provide the new repo details for your action
-
-## Code in Main
-
-> First, you'll need to have a reasonably modern version of `node` handy. This won't work with versions older than 9, for instance.
-
-Install the dependencies  
-```bash
-$ npm install
-```
-
-Build the typescript and package it for distribution
-```bash
-$ npm run build && npm run package
-```
-
-Run the tests :heavy_check_mark:  
-```bash
-$ npm test
-
- PASS  ./index.test.js
-  ✓ throws invalid number (3ms)
-  ✓ wait 500 ms (504ms)
-  ✓ test runs (95ms)
-
-...
-```
-
-## Change action.yml
-
-The action.yml contains defines the inputs and output for your action.
-
-Update the action.yml with your name, description, inputs and outputs for your action.
-
-See the [documentation](https://help.github.com/en/articles/metadata-syntax-for-github-actions)
-
-## Change the Code
-
-Most toolkit and CI/CD operations involve async operations so the action is run in an async function.
-
-```javascript
-import * as core from '@actions/core';
-...
-
-async function run() {
-  try { 
-      ...
-  } 
-  catch (error) {
-    core.setFailed(error.message);
-  }
-}
-
-run()
-```
-
-See the [toolkit documentation](https://github.com/actions/toolkit/blob/master/README.md#packages) for the various packages.
-
-## Publish to a distribution branch
-
-Actions are run from GitHub repos so we will checkin the packed dist folder. 
-
-Then run [ncc](https://github.com/zeit/ncc) and push the results:
-```bash
-$ npm run package
-$ git add dist
-$ git commit -a -m "prod dependencies"
-$ git push origin releases/v1
+```yaml
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: takescoop/terraform-register-module-action@v1
+        with:
+          repo: "{{ github.repository }}"
+          tf-organization: takescoop
+          tf-token: "{{ secrets.TF_TOKEN }}"
+          vcs-name: Github
```

Comment:
This should be the default, no? Would say this doesn't even need to be customizable but other editions of GH (e.g. Enterprise Server) might have distinct names.

Reply:
It doesn't appear to be an actual default, although there is a suggestion
![image](https://user-images.githubusercontent.com/8100360/125850753-84449182-73a3-4c2b-bfcd-2718746cb0d8.png)

I figured it was something like this because my cloud account uses the suggested `github.com` but we use `GitHub`

Reply:
We can default to `GitHub.com` for now though, seems reasonable enough